src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -377,6 +377,10 @@ Engine::LoadExeNetworkImpl(const InferenceEngine::CNNNetwork &network, const std
     }
 
     auto config = orig_config;
+    auto cache_dir = GetCore()->GetConfig(GetName(), ov::cache_dir.name());
+    if (cache_dir != ov::Any("")) {
+        config[ov::cache_dir.name()] = cache_dir.as<std::string>();
+    }
```

Comment:
CPU should remove explicit handling of cache dir https://github.com/openvinotoolkit/openvino/pull/16132

Reply:
ok, will update after #16132 is merged